### ISSUE TITLE
Manjaro keychain not allowing past "Verify your PIN code" onboarding step

### ISSUE THREAD

**Issue by fijter** [iotaledger/firefly#243]
## Bug description

On Linux while onboarding a Error is thrown in the console when I click `Set PIN code`; nothing happens in the UI and the console shows 

```
Error occurred in handler for 'keychain-set': [Error: Unknown or unsupported transport “disabled” for address “disabled:”]
```

## wallet version

Which version of wallet are you using?

- wallet version (version number, commit, or branch): 0.0.1 release (internal build shared by Charlie)

## Hardware specification

What hardware are you using?

- Operating system: Arch Linux / Manjaro latest (BSPWM)
- RAM: 32GB
- Cores: 8

## Steps To reproduce the bug

Explain how the maintainer can reproduce the bug.

1. Run a fresh copy of Firefly on Linux/Manjaro
2. Follow the onboarding steps until you get to the Verify your PIN code screen
3. Click the Set PIN code button

## Expected behaviour

The next step in the onboarding

## Actual behaviour

Nothing happens

## Errors

```Error occurred in handler for 'keychain-set': [Error: Unknown or unsupported transport “disabled” for address “disabled:”]```

Note: Maybe it has to do with a keychain dependancy that is not checked on OS level? I had issues with launching Trinity as well.

**Comment by whig0** [comment]
I have the same problem on KDE Neon (latest). Error message in console is:
`Error occurred in handler for 'keychain-set': [Error: The name org.freedesktop.secrets was not provided by any .service files]`

**Comment by taro-k** [comment]
Here, Debian 10 (4.19.0-6-amd64) and installed all of:
```
libsecret-1-0
libsecret-1-dev
libsecret-common
libsecret-tools                 
gnome-keyring
```
but still have same error..

**Comment by Schurick** [comment]
Had same issue with Manjaro (KDE). The problem in my case was, that I executed the AppImage as admin. After being sure that libsecret and gnome-keyring is installed, I setup udev rules as described in the link below. After that I executed the AppImage with user rights and could pass the pin-screen. Maybe that helps.

[https://support.ledger.com/hc/en-us/articles/115005165269-Fix-USB-connection-issues-with-Ledger-Live?support=true]

PS: I also closed kwallet deamon before that, but can not say afterwords if that was really necessary.
